AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -269,6 +269,7 @@ flashff-blog.com##.rsswaku
 momoniji.com##div[class*="pick-Box"]
 tousatu-club.com##.widget_custom_html
 eroanime-movie.com##.rss-grid
+artstation.com##.s-grid
```

Comment:
It's social share button, so please add rule to - https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt

I also would like something more related to "share" button if possible, so something like:
```adblock
artstation.com##.share-grid > .s-grid
```
or
```adblock
artstation.com##body .share-grid
```
should be fine.

Just:
```adblock
artstation.com##.share-grid
```
doesn't work, because website's style has higher specificity/priority.